scripts/jenkins/qat/Makefile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Makefile for QAT-DPDK jenkins.
+
+qat: qat-checks qat-pull qat-plugin qat-tests
+
+qat-checks:
+	bash checks.sh
+
+qat-pull:
+	bash images-pull.sh
+
+qat-plugin:
+	bash plugin-deploy.sh
+
+qat-tests: qat-tc-crypto qat-tc-compress
```

Comment:
no need to split between crypto and compress. I'd prefer you just loop all `test-*`. Then the makefile does not need to change if a new case is acced.